package.json

```diff
@@ -6,8 +6,12 @@
   "scripts": {
     "start": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "lint": "eslint -c .eslintrc.cjs --ext .ts src",
+    "check:code": "eslint -c .eslintrc.cjs src",
+    "fix:code": "npm run check:code -- --fix",
+    "pre-commit:fix:code": "eslint -c .eslintrc.cjs --fix",
```

Comment:
Is prettier already considered in the eslint linting process? If not, we'll need to also use it and add it to the pre-commit hook so the code gets prettified as well.